stubs/pyinstaller/pyi_splash/__init__.pyi

```diff
@@ -0,0 +1,12 @@
+# Referenced in: https://pyinstaller.org/en/stable/advanced-topics.html#module-pyi_splash
+# Source: https://github.com/pyinstaller/pyinstaller/blob/develop/PyInstaller/fake-modules/pyi_splash.py
+from typing_extensions import Literal
+
+__all__ = ["CLOSE_CONNECTION", "FLUSH_CHARACTER", "is_alive", "close", "update_text"]
+
+def is_alive() -> bool: ...
+def update_text(msg: str) -> None: ...
+def close() -> None: ...
+
+CLOSE_CONNECTION: Literal[b"\u0004"]
+FLUSH_CHARACTER: Literal[b"\r"]
```

Comment:
Is this module in the wrong directory? Shouldn't it be `stubs/pyinstaller/PyInstaller/pyi_splash/__init__.pyi`, rather than `stubs/pyinstaller/pyi_splash/__init__.pyi`?

Reply:
You use it as such: `import pyi_splash` (see PR description for a code snippet). Which means it's a different `*-stubs` than PyInstaller. (which I've also tested by manually adding `pyi_splash-stubs` to my site-packages folder)

My current understanding is that with this setup, typeshed will publish a single `pyinstaller-types` package to PyPI. Which when installed will add both `PyInstaller-stubs` and `pyi_splash-stubs`.

Reply:
Huh. I've never seen it done before. I'd want to make sure the typeshed bots would do the right thing when uploading the package to PyPI.

Sadly I know piteously little about how the logic in https://github.com/typeshed-internal/stub_uploader works. Can @hauntsaninja or @srittau help, maybe? :)

Reply:
Should work just fine!

Reply:
https://github.com/typeshed-internal/stub_uploader/blob/a4aba444e8bc45eb583efaeff268f04e4e976a19/stub_uploader/build_wheel.py#L154

Reply:
Awesome, thank you!